bluemira/geometry/tools.py

```diff
@@ -222,11 +222,68 @@ def make_polygon(
     -------
     wire: BluemiraWire
         a bluemira wire that contains the polygon
+
+    Notes
+    -----
+    If the input points are closed, but closed is True, the returned BluemiraWire will be
```

Comment:
```suggestion
    If the input points are closed, but closed is False, the returned BluemiraWire will be
```

Reply:
> A couple of docs changes I think.
> 
> The additions to `make_polygon` and `make_bezier` in tools.py feel like wrapper operations that might be more suited to a decorator of the functions?

I've just got 90% of the way on a wrapper, but realising now that the behaviour for the two functions is subtly different (for make_polygon we have to drop the closure point and for interpolate_bspline we don't). 

Don't merge yet I'm adding a decorator with an arg and probably a test

Reply:
OK done